components/link/css/story-template.jsx

```diff
@@ -62,12 +62,16 @@ export const defaultArgs = {
   focusVisible: false,
   href: null,
   telephone: false,
-  textContent: 'Voorbeeld van een link',
+  textContent: '',
   title: '',
   visited: false,
   iconleft: false,
 };
 
+export const exampleArgs = {
+  textContent: 'Voorbeeldlink',
+};
+
 export const Link = ({
```

Comment:
`aria-current={current ?'page' : null}` or else every link will have this `aria-current="false"`